idl/cherami_server/admin.thrift

```diff
@@ -222,3 +222,17 @@ service ControllerHostAdmin {
    */
   void extentsUnreachable(1: ExtentsUnreachableRequest request)
 }
+
+struct ReplicatorConnection {
+    1: optional string extentUUID
+    2: optional i64 startTime
```

Comment:
More fields of interest: last activity time, number of messages/bytes transferred.